internal/commands/push.go

```diff
@@ -306,3 +310,16 @@ func (r *plainDisplay) onEvent(ev remotes.FixupEvent) {
 		}
 	}
 }
+
+func checkFlags(flags *pflag.FlagSet) error {
+	setFlags := 0
+	flags.Visit(func(f *pflag.Flag) {
```

Comment:
I think that you can just check if the flag has been modified using https://godoc.org/github.com/spf13/pflag#FlagSet.Changed